integration-tests/pom.xml

```diff
@@ -32,8 +32,20 @@
 
     <name>Camel Quarkus :: Integration Tests</name>
 
+    <properties>
+        <!--
+            Make quarkus use docker by default as native-image does not
+            work anymore due to a Fedora upgrade
+
+            See https://github.com/oracle/graal/issues/1582
+         -->
+        <native-image.docker-build>true</native-image.docker-build>
+        <native-image.container-runtime>docker</native-image.container-runtime>
```

Comment:
Could you please adapt the contributor guide? https://github.com/apache/camel-quarkus/blob/35075434992f268f7865cfebcb9854290827437c/docs/modules/ROOT/pages/contributor-guide.adoc